services/github/github-search.tester.js

```diff
@@ -8,4 +8,4 @@ t.create('hit counter')
 
 t.create('hit counter for nonexistent repo')
   .get('/badges/puppets/async%20handle.json')
-  .expectBadge({ label: 'counter', message: 'repo not found' })
+  .expectBadge({ label: 'async handle counter', message: '0' })
```

Comment:
This query no longer throws a 404 for repo doesn't exist - it just says zero results

Reply:
Well, this is unfortunate.